### PR TITLE
code review

### PR DESCRIPTION
**Pull request by alexisintech**
This PR

- removes the "Allowlist" page, in favor of this new "Restrictions" page
- updates the images to be full screen images, providing more context to users. these images were optimized using squoosh. these images were placed in a folder that has a more clear directory path that reflects the page the images can be found on.
- adds an "overview" introduction to the content of the page
- updates navigation references to be more clear, such as:
"Enable this feature by navigating to the **Allowlist** section in the **Restrictions** tab under the **User & Authentication** section." 
instead of 
"Enable this feature by navigating to the [Restrictions](https://dashboard.clerk.com/last-active?path=/user-authentication/signup-restrictions) screen in the Clerk Dashboard."
- fixes heading structure accordingly with a11y (one h1 for a page, followed by sequential headings - h2, h3, etc).

### CODE REVIEW

File: docs/authentication/configuration/restrictions.mdx [hunk 1]
```diff
@@ -1,54 +1,76 @@
...
-sanity_slug: /docs/authentication/restrictions
+title: Restrictions | Authentication
+description: Learn how to manage user access to your application by configuring Clerk's access restrictions.
 ---
 
-# Overview
+# Restrictions
 
-# Allowlist
+Clerk provides a set of restriction options designed to provide you with enhanced control over who can gain access to your application. Through these restriction settings, you can limit sign-ups or prevent accounts with specific identifiers from accessing your application.  These identifiers can be email addresses, phone numbers, and even entire domains. To get started, navigate to the [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions) tab under the **User & Authentication** section of the [Clerk Dashboard](https://dashboard.clerk.com).
```
Comment (fragoulis) [inline docs/authentication/configuration/restrictions.mdx:8]: There are 2 spaces before `These identifiers ...`

Comment (fragoulis) [reply docs/authentication/configuration/restrictions.mdx:8]: You don't need a comma `,` before `and` : `phone numbers, and ...`

**Review by gkats — APPROVED** [review]
Massive improvement and very detailed explanations. Thank you @fragoulis and @alexisintech for taking over and moving this to another level. 

:100:

File: docs/authentication/configuration/restrictions.mdx (hunk 1 above)
Comment (alexisintech) [reply docs/authentication/configuration/restrictions.mdx:8]: > You don't need a comma `,` before `and` : `phone numbers, and ...`

actually, you do; its called an oxford comma!

Comment (gkats) [reply docs/authentication/configuration/restrictions.mdx:8]: :nerd_face: TIL

**Pull request merged**